src/routes/index.ts

```diff
@@ -61,4 +68,58 @@ Disallow: /`);
     req.log.debug({ body: req.body }, 'allocation received');
     return res.status(204).send();
   });
+
+  fastify.post<{ Body: { userIds: string[] } }>(
+    '/digest/send',
+    async (req, res) => {
+      // TODO AS-122-email-schedule-endpoint authenticate here
+
```

Comment:
Not sure if we already authenticate somehow from retool? I know we have automations part but that is more on exposing automation stuff from API, this would be pinging API endpoint from retool. As mentioned in RFC I though of just adding a secret that we can send from retool?

Reply:
As addition we can also limit by IP ranges https://docs.retool.com/data-sources/concepts/ip-allowlist-cloud-orgs, again not sure if we connected something already from retool.

Reply:
I added bearer auth here 6ec050347f5759a4a60116d63999fd31ee80f4df